File: src/main/java/guru/xgm/language/jass/openapi/actionSystem/convert/jass2lua/Jass2LuaAction.java
package guru.xgm.language.jass.openapi.actionSystem.convert.jass2lua;

import guru.xgm.language.jass.openapi.actionSystem.convert.Jass2AnyAction;
import guru.xgm.language.jass.openapi.actionSystem.convert.Jass2AnyVisitor;
import org.jetbrains.annotations.NotNull;

public class Jass2LuaAction extends Jass2AnyAction {

    public Jass2LuaAction() {
        super();
    }

    @Override
    public @NotNull Jass2AnyVisitor getVisitor() {
        return new Jass2LuaVisitor();
    }

    @Override
    public @NotNull String getTargetExtension() {
        return ".lua";
    }
}

File: src/main/kotlin/guru/xgm/language/jass/openapi/actionSystem/convert/jass2lua/Jass2LuaAction.kt
package guru.xgm.language.jass.openapi.actionSystem.convert.jass2lua

import guru.xgm.language.jass.openapi.actionSystem.convert.Jass2AnyAction
import guru.xgm.language.jass.openapi.actionSystem.convert.Jass2AnyVisitor

class Jass2LuaAction : Jass2AnyAction() {
    override val visitor: Jass2AnyVisitor
        get() = Jass2LuaVisitor()

    override val targetExtension: String
        get() = "lua"
}
